dev_guide/application_health.adoc

```diff
@@ -44,6 +44,21 @@ Set a readiness check by configuring the
 
 |===
 
+The exact timing of a probe is controlled by two fields,
+both expressed in units of seconds:
+
+|====
+| Field | Description
+
+| `*initialDelaySeconds*`
+| How long to wait after the container starts to begin the probe.
+
+| `*timeoutSeconds*`
+| How long to wait for the probe to finish (default: `1`).
+If this time is exceeded, OpenShift considers the probe to have failed.
```

Comment:
This sentence is not wrong, but it may lead to misunderstanding. Someone may misunderstand that OpenShift restarts pods after the probe failed. In actuality, OpenShift restarts the pod when the number of timeout reached `failureThreshold` below.

https://github.com/kubernetes/kubernetes/blob/master/api/swagger-spec/v1.json#L13556-L13565

~~~
     "successThreshold": {
      "type": "integer",
      "format": "int32",
      "description": "Minimum consecutive successes for the probe to be considered successful after having failed. Defaults to 1. Must be 1 for liveness. Minimum value is 1."
     },
     "failureThreshold": {
      "type": "integer",
      "format": "int32",
      "description": "Minimum consecutive failures for the probe to be considered failed after having succeeded. Defaults to 3. Minimum value is 1."
     }
~~~